lib/game_teams_manager.rb

```diff
@@ -75,38 +75,36 @@ def highest_visitor_team
       gameteam.sum{|game1| game1.goals.to_i} / gameteam.count.to_f
     end
   end
-  
-end
-
-
-#   def lowest_visitor_team
-#     worst_away_team = away_games_by_team_id.min_by do |team_id, gameteam|
-#       gameteam.sum{|game1| game1.goals.to_i} / gameteam.count.to_f
-#     end.first
 
-#     Team.all.find{|team1| team1.team_id == worst_away_team}.teamname
-#   end
+  def lowest_visitor_team
```

Comment:
Looks good John! Now we just need to have a Zoom chat so we can start merging all these files